fuzzer/cascade/randomize/pickfpuop.py

```diff
@@ -49,24 +49,29 @@ def gen_fpufsm_instrs(fuzzerstate):
         assert fuzzerstate.design_has_fpu
         assert fuzzerstate.privilegestate.privstate == PrivilegeStateEnum.MACHINE
 
+    ret = None
```

Comment:
I'd rather say ret = [], otherwise in the future the check on `len(ret)` might be problematic